src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -850,7 +850,7 @@ private BoundExpression BindTupleExpression(TupleExpressionSyntax node, Diagnost
             {
                 bool disallowInferredNames = this.Compilation.LanguageVersion.DisallowInferredTupleElementNames();
 
-                tupleTypeOpt = TupleTypeSymbol.Create(node.Location, elements, locations, elementNames,
+                tupleTypeOpt = TupleTypeSymbol.Create(node.Location, elements.SelectAsArray(TypeMap.AsTypeSymbolWithAnnotations), locations, elementNames,
```

Comment:
>AsTypeSymbolWithAnnotations [](start = 100, length = 27)

Where are the annotations coming from?

Reply:
Changed to use `BoundExpressionExtensions.IsNullable()` to get annotations.